app/views/admin/administrators/administrators/delete.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Do you want to delete this user?</h1>
-<p>User: <%= @administrator.full_name %></p>
+<h3>User: <%= @administrator.full_name %></h3>
```

Comment:
@callumacrae am I right in thinking h1 and h3 without h2 is weird?

Reply:
We have the govuk-heading-s/m/l/xl classes

Reply:
Updated to govuk-!-font-weight-bold and govuk-!-font-weight-regular per Shani comments